widgets/DynamicSymbology/Widget.js

```diff
@@ -67,7 +67,7 @@ function(declare, BaseWidget, LayerInfos, dom, domConstruct, on, Map, Color, Col
 	  var curColor = new Color([92,92,92]);
 	  var dynamicSym = this;
 
-	  dynamicSymbology.isSmartMapping = false;
+	  dynamicSymbology.isSmartMapping = true;
```

Comment:
Maybe I've been staring at it for too long but I'm not really impressed so far. It's a quick way to product a nice renderer but it lacks control over some properties that I would assume most people would want access to.